src/ui/webeiddialog.cpp

```diff
@@ -581,7 +585,7 @@ void WebEidDialog::setupCertificateAndPinInfo(
     }
 }
 
-void WebEidDialog::setupPinPrompt(const PinInfo& pinInfo)
+void WebEidDialog::setupPinPrompt(PinInfo pinInfo)
```

Comment:
Because static analyser thinks it is small enough to pass as value
https://github.com/KDE/clazy/blob/1.11/docs/checks/README-function-args-by-value.md